tests/test_netstats.py

```diff
@@ -16,6 +16,31 @@
 logger.setLevel(50)
 
 
+@pytest.mark.parametrize("value", [True, 3.14, "word", {"project":"PyNets"}])
+def test_get_prop_type(value):
+    """
+    Test for get_prop_type() functionality
+    """
+    start_time = time.time()
+    tname, value, key = netstats.get_prop_type(value)
+    print("%s%s%s" % ('thresh_and_fit (Functional, proportional thresholding) --> finished: ',
+                      np.round(time.time() - start_time, 1), 's'))
+    assert tname is not None
```

Comment:
Can form more meaningful assert statement here given that we know tname will have to take on one of either: 
'bool', 'float', 'string', 'object'.